scripts/dts/gen_defines.py

```diff
@@ -518,6 +533,14 @@ def write_phandle_val_list_entry(dev, entry, i, ident):
         if i is not None:
             cell_ident += "_{}".format(i)
         out_dev(dev, cell_ident, val)
+        initializer_vals.append(val)
```

Comment:
Could move it outside the loop as well, like this:

```python
    ...
    out_dev(...)

initializer_vals += entry.specifier.values()

initializer_ident = ...
```

Pretty subjective though.